db/migrations/20210308070729-add-captain-to-group.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.bulkDelete('Shipments', {})
+    await queryInterface.bulkDelete('Groups', {})
```

Comment:
Question: why is this necessary?

Reply:
Eh, it probably isn't. Everyone is going to need to reset their DBs anyway. I'll remove it.